cfg/acq.cfg

```diff
@@ -117,6 +121,10 @@ radiant:
 			window=50;
 			//Number of coincidences (min 1) in this coincidence trigger
 			num_coincidences=2;
+			//Time delay (in ns) to delay readout of channels in group mask
+			readout_delay=587;
+			//Group mask of which channels will be delayed on this trigger
```

Comment:
can this be documented a bit more? It seems like it's 4 bits?

Reply:
It is 4 bits. Channels are broken up into 4 groups: surface, power string, and the 2 helper strings. I can write a more informative comment describing the bits in the mask